layouts/partner/aws.html

```diff
@@ -14,13 +14,26 @@ <h1>{{ .Params.hero.title }}</h1>
             <div class="w-full order-first lg:order-last mt-8 lg:mt-0 lg:w-1/2 lg:mt-0">
                 <div class="rounded shadow-md" style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden;">
                     <iframe
-                        src="//www.youtube.com/embed/cCzoCUxqiVo?rel=0"
+                        src="//www.youtube.com/embed/M8PEXcFh_aQ?rel=0"
                         style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;"
                         allowfullscreen=""
                         title="Pulumi + AWS: Modern Infrastructure as Code"
-                        srcdoc="<style>*{padding:0;margin:0;overflow:hidden}html,body{height:100%}img{position:absolute;width:100%;top:0;bottom:0;margin:auto}</style><a href=https://www.youtube.com/embed/cCzoCUxqiVo?autoplay=1><img src='/images/partners/aws-apn-video-thumbnail.png' alt='Pulumi + AWS: Modern Infrastructure as Code'></a>"
+                        srcdoc="<style>*{padding:0;margin:0;overflow:hidden}html,body{height:100%}img{position:absolute;width:100%;top:0;bottom:0;margin:auto}</style><a href=https://www.youtube.com/embed/M8PEXcFh_aQ?autoplay=1><img src='/images/partners/kube_interview_thumbnail.jpg' alt='Pulumi + AWS: Modern Infrastructure as Code'></a>"
                     ></iframe>
                 </div>
+                <p class="my-4 text-center w-full">
+                    <a class="inline-flex items-center text-white hover:text-white font-bold" href="{{ relref . "/case-studies/snowflake" }}">
+                        <span class="inline-block w-6 text-orange-300 mr-3">
+                            <svg aria-hidden="true" focusable="false" data-prefix="fad" data-icon="sparkles" class="svg-inline--fa fa-sparkles fa-w-16" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><g class="fa-group"><path class="fa-secondary" fill="currentColor" d="M423.16 186.58L448 127l59.58-24.84a8 8 0 0 0 0-14.32L448 63 423.16 3.42a8 8 0 0 0-14.32 0L384 63l-59.58 24.84a8 8 0 0 0 0 14.32L384 127l24.84 59.58a8 8 0 0 0 14.32 0zm-14.32 136.84L384 383l-59.58 24.84a8 8 0 0 0 0 14.32L384 447l24.84 59.58a8 8 0 0 0 14.32 0L448 447l59.58-24.84a8 8 0 0 0 0-14.32L448 383l-24.84-59.58a8 8 0 0 0-14.32 0z" opacity="0.4"></path><path class="fa-primary" fill="currentColor" d="M384 254.64a16.06 16.06 0 0 0-8.84-14.33l-112.57-56.39-56.28-112.77c-5.44-10.87-23.19-10.87-28.62 0l-56.28 112.77L8.84 240.31a16 16 0 0 0 0 28.67l112.57 56.39 56.28 112.77a16 16 0 0 0 28.62 0l56.28-112.77L375.16 269a16.07 16.07 0 0 0 8.84-14.36z"></path></g></svg>
+                        </span>
+                        <span class="mr-2 hover:mr-3" style="transition: all 100ms linear;">
```

Comment:
Can you not just use `transition-all`?

```suggestion
                        <span class="mr-2 hover:mr-3 transition-all">
```

Reply:
Yup can do. Updated the `/kubernetes` page too